src/serac/physics/contact/contact_data.cpp

```diff
@@ -328,12 +328,14 @@ void ContactData::updateDofOffsets() const
   offsets_up_to_date_ = true;
 }
 
+}  // namespace serac
```

Comment:
Are we sure this is right? the `namespace serac` is inside the if of the ifdef then reopens in the else

Reply:
Pretty sure it wasnt right to begin with either... Can you move all namespacing outside of the ifdef?

Reply:
im pretty sure its right but it is weird to have namespace serac in both ifdef blocks.

Reply:
fixed